knowagedao/src/main/java/it/eng/spagobi/tools/dataset/dao/SbiDataSetDAOImpl.java

```diff
@@ -264,33 +264,32 @@ public List<SbiDataSet> loadDataSets(String owner, Boolean includeOwned, Boolean
 			session = getSession();
 
 			// create statement
-			String statement = "from SbiDataSet h where h.active = ?";
+			String statement = "from SbiDataSet h where h.active = :active";
 			if (owner != null) {
-				String ownedCondition = includeOwned ? "h.owner = ?" : "h.owner != ?";
+				String ownedCondition = includeOwned ? "h.owner = :owner" : "h.owner != :owner";
```

Comment:
Is there a possibility to merge the duplicated if statements? The if where we add the placeholder and the if where we set the placeholder's value.